src/vue/components/panel.vue

```diff
@@ -50,30 +50,36 @@
     <div class="alpheios-panel__content">
       <div
           class="alpheios-panel__tab-panel alpheios-panel__content_no_top_padding alpheios-panel__tab-panel--fw alpheios-panel__tab__definitions"
-          v-if="$store.getters['ui/isActiveTab']('definitions')"
+          v-show="$store.getters['ui/isActiveTab']('definitions')"
           data-alpheios-ignore="all"
           >
         <div class="alpheios-lookup__panel">
           <lookup
               :name-base="`panel-defs`"
           />
         </div>
-        <div v-if="$store.getters['app/defDataReady']">
-          <div :key="definition.ID"
-               class="alpheios-panel__contentitem" v-for="definition in formattedShortDefinitions">
-            <shortdef :definition="definition" :languageCode="$store.state.app.languageCode"></shortdef>
-          </div>
-          <div class="alpheios-panel__contentitem alpheios-panel__contentitem-full-definitions"
-               v-html="formattedFullDefinitions"></div>
-        </div>
-        <div v-else>
```

Comment:
see previous quesiton

Reply:
Put it back back temporary until #389 is resolved.